py-polars/polars/__init__.py

```diff
@@ -414,3 +414,38 @@ def __getattr__(name: str) -> Any:
 
     msg = f"module {__name__!r} has no attribute {name!r}"
     raise AttributeError(msg)
+
+
+# fork() breaks Polars thread pool. Instead of silently hanging when users do
+# this, e.g. by using multiprocessing's footgun default setting on Linux, warn
+# them instead:
+def __install_postfork_hook() -> None:
```

Comment:
Nice fix. Any idea of the startup cost of this?

Reply:
I suspect it's not meaningfully slower, but I'll measure it tomorrow.

Reply:
This appears to add 3 or 4 µs to `import polars`. This doesn't seem like a problem to me.

Reply:
Alright. Thanks @itamarst. Nice that we can catch those deadlocks now.